carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/R.java

```diff
@@ -48,7 +48,9 @@
 
     DATABASE("database.properties"),
 
-    ZAFIRA("zafira.properties");
+    ZAFIRA("zafira.properties"),
+
+    CAPABILITIES("custom_capabilities.properties");
```

Comment:
@candelaripoli, we don't plan to use capabilities outside of config.properies. Please remove R.CAPABILITIES declaration and update your PR accordingly. thanks!